examples/trading/api/README.md

```diff
@@ -0,0 +1,120 @@
+# Escrow API Demo
+
+This demo is built to showcase how we can build an event indexer + API
+to easily & efficiently serve on-chain data for our app.
+
+The demo indexer uses polling to watch for new events.
+
+Everything is pre-configured on testnet, but can be easily tweaked to work on any other network.
```

Comment:
Can we add a brief explanation of how here?

Reply:
Good point, will do!